integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -310,7 +318,13 @@ resources:
 - name: alluxio-logs
   emptyDir:
     medium: {{ .Values.logserver.medium }}
+    {{- /*
+    Re-enable emptyDir sizeLimit after addressing the following git issue:
+    https://github.com/Alluxio/alluxio/issues/12277
+    */}}
+    {{- if ne .Values.logserver.medium "Memory" }}
     sizeLimit: {{ .Values.logserver.size | quote }}
+    {{- end }}
```

Comment:
Not sure how the logserver handles a memory-backed `emptyDir` volume but I disabled it to be safe. I can remove this if it's unnecessary.

Reply:
Actually I believe it _is_ unnecessary, I don't think Alluxio tries to do any mounting or anything with that volume.